tcg/tcg-op.c

```diff
@@ -2960,19 +2938,19 @@ void tcg_gen_qemu_st_i32_with_checked_addr(TCGv_i32 val, TCGv_cap_checked_ptr ad
 
     plugin_gen_mem_callbacks(addr, info);
 #if defined(TARGET_MIPS) || defined(TARGET_RISCV) || defined(CONFIG_TCG_LOG_INSTR)
```

Comment:
hmm this should probably be `defined(TARGET_CHERI)`

Reply:
gen_cheri_break_loadlink needs a tcop as well, even if not CHERI,

Reply:
yeah just noticed that, so we need `defined(TARGET_MIPS) || defined(TARGET_RISCV) || defined(TARGET_CHERI)`

Reply:
Removed the extra arg.